lib/6to5/generation/whitespace.js

```diff
@@ -11,15 +11,17 @@ Whitespace.prototype.getNewlinesBefore = function (node) {
   var startToken;
   var endToken;
   var tokens = this.tokens;
+  var token;
+
+  for (var i = 0; i < tokens.length; i++) {
```

Comment:
AFAIK it's not an issue with V8, although I may be wrong.

I'm mainly following this guide:
https://github.com/petkaantonov/bluebird/wiki/Optimization-killers